python/CMakeLists.txt

```diff
@@ -66,4 +67,5 @@ install(
 # Test
 install(
   CODE "file(INSTALL ${CMAKE_CURRENT_BINARY_DIR}/test/test_binding.py DESTINATION \"${CMAKE_INSTALL_PREFIX}/python\")"
+  CODE "file(INSTALL ${CMAKE_CURRENT_BINARY_DIR}/test/test_api.py DESTINATION \"${CMAKE_INSTALL_PREFIX}/python\")"
```

Comment:
My assumption that `install` method of `CMake` would be better to install files if it works: 
```CMake
install(
  FILES 
  ${CMAKE_CURRENT_BINARY_DIR}/test/test_api.py
  ${CMAKE_CURRENT_BINARY_DIR}/test/test_binding.py
  DESTINATION ${CMAKE_INSTALL_PREFIX}/python
  )
  ```